app/views/layouts/_header.html.erb

```diff
@@ -35,3 +35,12 @@
   </div>
 </nav>
 
+<style>
+  @media (max-width: 767px){
+    .navbar{
+      max-height: 100vh;
+      overflow-y: auto;
+      -websit-overflow-scrolling:touch;
+    }
```

Comment:
@PeculiarE Can we also do without this rule? Does it make a difference?

Reply:
We can leave it in then. Thank you!!

Reply:
Alright 😄...I should be thanking you for helping figure out the container padding. So thank you ❤️